src/utils/ucc_rcache.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_RCACHE_H_
+#define UCC_RCACHE_H_
+
+#include <ucs/memory/rcache.h>
+#include <ucm/api/ucm.h>
+
+#define ucc_rcache_t                 ucs_rcache_t
```

Comment:
general comment regarding "rcache". We need to support "EXTERNAL_EVENTS" functionality (similar to hcoll) when UCX hooks are overwritten. we can add a todo.